WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -240,8 +240,10 @@ Math.trunc = Math.trunc || function(x) {
 
 // Setting up bootstrap-table
 $(function() {
+  $('table[data-toggle="table"]').addClass('bootstrap-table');
```

Comment:
Ok, what about options? Should we set them via data attrbitues or when initializing?